var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -183,3 +183,14 @@ def cmake_args(self):
                 args.append("-DENABLE_STATIC_LIBS=NO")
 
         return args
+
+    def test_ptls(self):
+        """
+        Run parseThat on /bin/ls to rewrite with basic instrumentation
+        """
```

Comment:
Thanks for the changes.  Formatting the docstring this way yields:
```
==> Spack test dyninst
==> Testing package dyninst-12.3.0-qp5gcub
==> [2023-05-24-15:11:53.660958] test: test_ptls: 
        Run parseThat on /bin/ls to rewrite with basic instrumentation
        
==> [2023-05-24-15:11:53.676657] '/usr/WS1/dahlgren/releases/spack/opt/spack/linux-rhel8-broadwell/gcc-10.3.1/dyninst-12.3.0-qp5gcub7y4svxcphngown7gp3inafdik/bin/parseThat' '--binary-edit=/g/g21/dahlgren/.spack/test/j7dex34frx7y5qwev4v3rno2k4pqrney/ls.rewritten' '/bin/ls'
...
```

Would you mind tightening it up a little?

```suggestion
        """Run parseThat on /bin/ls to rewrite with basic instrumentation"""
```

Reply:
Fixed.

Reply:
Thanks!